canonical-webteam/docker-bundler
# Add extra compilation libraries for building e.g. nokogiri

QA
--

Build this image and name it `bundler`:

``` bash
docker build --tag bundler .
```

Now clone my new branch of vanillaframework.io, try to run it, see it fail:

``` bash
$ git clone -b local-dev-tooling https://github.com/nottrobin/vanillaframework.io
$ cd vanillaframework.io
$ ./run
...
An error occurred while installing nokogiri (1.7.2), and Bundler cannot continue.
Make sure that `gem install nokogiri -v '1.7.2'` succeeds before bundling.
```

Edit the `./run` script to use the image we just built:

``` bash
# Define the versions of the two docker images
#bundler_image="canonicalwebteam/bundler:v0.1.0"
bundler_image="bundler"
```

Now run the site, see that ruby dependencies install correctly:

``` bash
$ ./run
...
Bundle complete! 3 Gemfile dependencies, 79 gems now installed.
```
